Description: remove reference to online CSS, privacy problem
Author: W. Martin Borgert <debacle@debian.org>
Origin: vendor
Last-Update: 2018-08-11
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -54,7 +54,6 @@
 
 html_context = {
     'extra_css_files': [
-        '//quodlibet.github.io/fonts/font-mfizz.css',
         '_static/extra.css',
     ],
 }
